oneapp-server/types/ApplicationContact.js

```diff
@@ -0,0 +1,91 @@
+const { gql } = require('apollo-server-express');
+
+const typeDef = gql`
+  extend type Application {
+    "The individual applying for benefits"
+    contact: ApplicationContact
+  }
+
+  type ApplicationContact {
+    "Applicant first name"
+    APPLICANT_FIRST_NAME: String
+    "Applicant last name"
+    APPLICANT_LAST_NAME: String
+    "Applicant middle name"
+    APPLICANT_MIDDLE_NAME: String
+    "Application maiden name"
+    APPLICANT_MAIDEN_NAME: String
+    "Payee first name, if different than applicant"
+    PAYEE_FIRST_NAME: String
+    "Payee last name, if different than applicant"
+    PAYEE_LAST_NAME: String
+    "Is applicant homeless"
+    IS_HOMELESS: Boolean
+    "Applicant address line 1"
+    ADDRESS1: String
+    "Applicant address line 2"
+    ADDRESS2: String
+    "Applicant address city"
+    CITY: String
+    "Applicant address state"
+    STATE: String
+    "Applicant address zipcode"
+    ZIP: Int
+    "Applicant address zipcode extend 4"
+    ZIP4: Int
+    "New Jersey county id, retrieved from zipcode to county lookup"
+    COUNTY_NUMBER: Int
+    "Mailing address line 1"
+    M_ADDRESS1: String
+    "Mailing address line 2"
+    M_ADDRESS2: String
+    "Mailing address city"
+    M_CITY: String
+    "Mailing address state"
+    M_STATE: String
+    "Mailing address zipcode"
+    M_ZIP: Int
+    "Mailing address zipcode extended 4"
+    M_ZIP4: Int
+    "Applicant home phone number"
+    HOME_PHONE_NUMBER: Int
+    "Applicant work phone number"
+    WORK_PHONE_NUMBER: Int
+    "Applicant cell phone number"
+    CELL_PHONE_NUMBER: Int
+    "Applicant other phone number"
+    OTHER_PHONE_NUMBER: Int
+    "Applicant email address"
+    EMAIL_ADDRESS: String
+    "Does applicant not have a phone number"
+    NO_PHONE_NUMBER: Boolean
+    "Does applicant not have contact information"
+    NO_CONTACT_INFORMATION: Boolean
+    "The source of the application; always OA"
+    APPLICATION_TYPE: ApplicationType
+    "Is special New Jersey Help application; can be null"
+    APPLICATION_SUBTYPE: ApplicationSubtype
```

Comment:
Unsure if we want to persist this to the front-end end or this is a field the back-end will handle behind the scenes. The ApplicationType will always be `OA`. From my conversations with Carmen, I do not think ApplicationSubType is used anymore.

Reply:
do we want to shift the type/subtype fields to the Application data obj? as of now, looks as though the Application data obj just has the user ID?

Reply:
From a data structure perspective, I would agree, it would make sense to move the fields to the Application data object. However, the existing schema defines both the fields as part of the `OA_RT_APP_CONTACT` type in the database, so that information is persisted alongside the contact information.

For maintainability for NJ, we're aiming to use all of their existing stored procedures, so they're familiar with the structure, as they've been making their business logic changes in the stored procedures. For compatibility sake, I think it makes sense to leave this on the contact, so we don't have to perform a separate lookup or modify existing stored procedures to return the fields on the Application type.